app/static/js/script.js

```diff
@@ -1,33 +1,53 @@
+import $ from "https://cdn.skypack.dev/jquery@3.6.0";
+//Accessiblity Feature - Dark/HighvContrast Mode
 document.addEventListener('DOMContentLoaded', function () {
   var modeSwitch = document.querySelector('.mode-switch');
+});
 
-  modeSwitch.addEventListener('click', function () {                     document.documentElement.classList.toggle('dark');
-    modeSwitch.classList.toggle('active');
+function confirmLogOutIntent() {
+  $('a#logoutModal').click(function () {
+    if (confirm('Are you sure to logout')) {
+      return true;
+    }
+    return false;
   });
-  
-  var listView = document.querySelector('.list-view');
-  var gridView = document.querySelector('.grid-view');
-  var projectsList = document.querySelector('.project-boxes');
-  
-  listView.addEventListener('click', function () {
-    gridView.classList.remove('active');
-    listView.classList.add('active');
-    projectsList.classList.remove('jsGridView');
-    projectsList.classList.add('jsListView');
-  });
-  
-  gridView.addEventListener('click', function () {
-    gridView.classList.add('active');
-    listView.classList.remove('active');
-    projectsList.classList.remove('jsListView');
-    projectsList.classList.add('jsGridView');
-  });
-  
-  document.querySelector('.messages-btn').addEventListener('click', function () {
-    document.querySelector('.messages-section').classList.add('show');
-  });
-  
-  document.querySelector('.messages-close').addEventListener('click', function() {
-    document.querySelector('.messages-section').classList.remove('show');
-  });
-});
\ No newline at end of file
+}
+
+modeSwitch.addEventListener('click', function () {
+  document.documentElement.classList.toggle('dark'); modeSwitch.classList.toggle('active');
+});
+
+//Enabling ToolTips Site Wide
+const tooltipTriggerList = [].slice.call(document.querySelectorAll('[data-bs-toggle="tooltip"]'));
+const tooltipList = tooltipTriggerList.map(function (tooltipTriggerEl) {
+  return new bootstrap.Tooltip(tooltipTriggerEl);
+});
+$(function () {
+  $('[data-toggle="tooltip"]').tooltip();
+});
+
+function hider(element_id) {
+  document.getElementById(element_id).setAttribute("hidden", "");
+}
+function shower(element_id) {
+  document.getElementById(element_id).removeAttribute("hidden", "");
+}
+
+function hooverStart() {
+  const element = document.getElementById('No-Password');
+  element.addEventListener('onmouseover',hider(element));
+}
+function hooverEnd(){
+  const element = document.getElementById('No-Password');
+  element.addEventListener('onmouseleave', hider(element));
+
+}
+
+function getDate(){
+  let currentYear = new Date().getFullYear(); 
+  return currentYear
+}
+
+  function ReCapatchaonSubmit(token) {
```

Comment:
Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/Terry-BrooksJr/animated-chainsaw/pullRequest?prid=9263623)
```suggestion
function ReCapatchaonSubmit(token) {
```